### PR TITLE
[zh-cn] Add kubectl_edit/_index.md

### PR DESCRIPTION
**Pull request by my-git9**
Add content/zh-cn/docs/reference/kubectl/generated/kubectl_edit/_index.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_edit/_index.md [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+<td colspan="2">--subresource string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+If specified, edit will operate on the subresource of the requested object. Must be one of [status]. This flag is beta and may change in the future.
+-->
+如果指定，edit 将对所请求对象的子资源进行操作。
+必须是 [status] 中的子资源之一。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_edit/_index.md:238]:
```suggestion
（目前）只能是 status 子资源。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_edit/_index.md [hunk 2]
```diff
@@ -0,0 +1,739 @@
...
+该标志是测试版，将来可能会发生变化。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_edit/_index.md:239]:
```suggestion
该标志处于 Beta 状态，将来可能会发生变化。
```

**Review by ydFu — APPROVED** [review]
/lgtm
/approve

**Pull request merged**